llvm/unittests/Support/TrailingObjectsTest.cpp

```diff
@@ -49,9 +50,9 @@ class Class1 final : protected TrailingObjects<Class1, short> {
   using TrailingObjects::getTrailingObjects;
 };
 
-// Here, there are two singular optional object types appended.  Note
-// that the alignment of Class2 is automatically increased to account
-// for the alignment requirements of the trailing objects.
+// Here, there are two singular optional object types appended. Note that the
```

Comment:
Looks like this and some other comment wrapping is unrelated change? Could you remove these? (if there's a desire to fix existing problematic formatting, you can send it separately - though we tend to avoid formatting-for-the-sake-of-formatting (except in cases where a file is being substantially changed, where having it be fully formatted to start would avoid new functional changes introducing major formatting changes too, etc)

Reply:
Done.